docs/how-to/no-backup.rst

```diff
@@ -22,3 +24,21 @@ Usage example
         Upgrade plan for 'rabbitmq-server' to victoria
             Upgrade software packages of 'rabbitmq-server' from the current APT repositories
         …
+
+Upgrade:
+
+.. code:: 
+
+    $ cou plan --no-backup --no-
```

Comment:
This line might be wrong. Maybe it's `cou upgrade --no-backup --no-interactive`

Reply:
The change is still using plan

Reply:
Good catch. Thanks